.coveragerc

```diff
@@ -0,0 +1,47 @@
+# .coveragerc for edx-platform
+[run]
+data_file = reports/.coverage
+source =
+    cms
+    common/djangoapps
+    common/lib/calc
+    common/lib/capa
+    common/lib/xmodule
+    lms
+    openedx/core/djangoapps
+
+omit =
+    cms/envs/*
+    cms/manage.py
+    cms/djangoapps/*/migrations/*
+    cms/djangoapps/*/features/*
+    lms/envs/*
+    lms/djangoapps/*/migrations/*
+    lms/djangoapps/*/features/*
+    common/djangoapps/terrain/*
+    common/djangoapps/*/migrations/*
+    openedx/core/djangoapps/*/migrations/*
+
+branch = true
+
+[report]
+ignore_errors = True
+
+exclude_lines =
```

Comment:
This one, on the other hand I do advocate that we keep. We don't need unit tests for purely defensive coding blocks. And developers should be able to use the no cover pragma.